src/cache/disk.rs

```diff
@@ -139,6 +142,7 @@ impl DiskCache {
         self.root.join("urls").join(encoded)
     }
 
+    /// Get the path to the cloned git repo
```

Comment:
If the comment is redundant with the function name, you can `#![allow(missing_docs)]` rather than repetitive comments